config/sillytips.txt

```diff
@@ -9,7 +9,6 @@ Completing your objectives is good practice, but the best antagonists will striv
 The more obscure and underused a game mechanic is, the less likely your victims are to be able to deal with it.
 Space is cold and it will quickly freeze you to death if you don't protect yourself. This isn't how thermodynamics really works but just go with it.
 Blobs are weak to fire! Use a flame thrower for maximum damage!
-Flash bangs can weaken blob tiles, allowing for you to easily destroy them.
```

Comment:
This was in silly tips for a reason.

The whole point was to not be nice about it, to be an anti-tip that would cause more harm.

if you want the game server's tips to be updated to reflect whats here you should leave this as is.